unity-renderer/Assets/DCLPlugins/ECS7/ComponentCrdtWriteSystem/ComponentCrdtWriteSystem.cs

```diff
@@ -21,9 +21,9 @@ private class MessageData
     private readonly ISceneController sceneController;
     private readonly IWorldState worldState;
 
-    private readonly Dictionary<string, CRDTProtocol> outgoingCrdt = new Dictionary<string, CRDTProtocol>(24);
-    private readonly Queue<MessageData> queuedMessages = new Queue<MessageData>(24);
-    private readonly Queue<MessageData> messagesPool = new Queue<MessageData>(24);
+    private readonly Dictionary<string, CRDTProtocol> outgoingCrdt = new Dictionary<string, CRDTProtocol>(60);
```

Comment:
Can we make these numbers constant to specify what they are?

Reply:
It's just a number calculated on top of my head to reduce the possibility of the dictionary resizing.
The number could be a lot lower or a lot higher depending on the amount of loaded scenes.
So adding a constant would generate more confusion than clarity